### ISSUE TITLE
Unknown import errors

### ISSUE THREAD

**Issue by jcs090218** [sstokic-tgm/JFTSE#166]
I'm trying to build the server from scratch today, but I got several errors:

```
import com.jftse.proto.auth.ClientServiceGrpc;
import com.jftse.proto.auth.FTClientList;
```

These two imports are missing and cannot be found. Any help is appreciated!

**Comment by sstokic-tgm** [comment]
Did you run `mvn clean install` in the root folder?
If not, cd into commons-proto and run there `mvn clean install` or `mvn clean package`

**Comment by jcs090218** [comment]
> Did you run `mvn clean install` in the root folder?

Yes, I did. I followed the instruction from the README but unfortunately it doesn't work. :(

> If not, cd into commons-proto and run there `mvn clean install` or `mvn clean package`

Still not working.

![2023-09-05 16 46 35](https://github.com/sstokic-tgm/JFTSE/assets/8685505/3764b8b3-7fba-4731-9de8-f4bd54780464)

**Comment by sstokic-tgm** [comment]
Does it still happen? I am quite unsure how to fix it, because the import errors should be gone with a project whole compile. Maybe you can try to just use mvn compile inside commons-proto

**Comment by jcs090218** [comment]
Everything seems to be built successfully in the command line. However, it doesn't work in Intellij. 🤔 

> Maybe you can try to just use mvn compile inside commons-proto

Unfortunately, I still get the same result. :(

**Comment by sstokic-tgm** [comment]
A few days ago I had the same problem. Within IntelliJ using the maven plugin, I started then the `clean` task and then the `install` task on commons-proto module. Afterwards I ran the auth-server with the play button (for the service so to say) and it worked.
